app/frontend/actions.ts

```diff
@@ -1255,89 +1255,75 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     })
   }
 
-  /* Pool Owner */
-
-  const deserializeTransactionFile = (file) => {
-    if (!file || !file.cborHex) {
-      throw NamedError('PoolRegInvalidFileFormat')
-    }
-
-    const unsignedTxDecoded = decode(file.cborHex)
-    const parsedTx = parseUnsignedTx(unsignedTxDecoded)
-    return parsedTx
-  }
+  /* POOL OWNER */
 
-  const loadPoolCertificateTx = async (state: State, fileObj) => {
+  const loadPoolCertificateTx = async (state: State, file: string) => {
```

Comment:
I'd rename file to fileContent or fileContentStr to make it clearer what it is or what the string actually is (my first guess was file name)